### DISCUSSION TITLE
No Media to Process - YouTube Private video

### DISCUSSION THREAD

**Discussion by tangosdad** [aclap-dev/video-downloadhelper#501] [Help and Q&A]
### Your question

I am unable to download YouTube private videos. No media to process

### Addon & CoApp version and details

version: 8.2.2.8a1
channel: beta
build date: 2024-03-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 21a04c354c224de6
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Mark",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

Does VDH 9.0.1.13a1 beta work on your content?  VDH has had a problem downloading private YouTube content for a few years.  I don't know if that issue has been addressed.  But you should be using the latest beta anyway.  Try it & let us know how you get on.

**Reply by tangosdad** [reply]
Well, that was a good idea, but unfortunately, I installed the beta and still can not download a private YouTube video. But thank you for the feedback and idea anyway!

**Reply by Mattmjs** [reply]
When you say private, is it unlisted on YouTube or does it require a password ? And no detection correct ?
Sounds similar in that there is no detection in google drive video.

**Reply by tangosdad** [reply]
It is a video that has been shared with me and I have to be logged into my Google/YouTube account. Does not get detected by Video DownloadHelper

**Reply by Mattmjs** [reply]
Could you share the network activity in the developer tools then attach the file here. Go to the private video and make sure it is
empty in the network then reload it again. Then follow what the image shows below.

![3-10-23](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/16b2867a-3aae-45b4-8bdb-e3d5c794faea)

**Comment by Mattmjs** [comment]
I put up a video and made it private on YouTube, it looks like they don't make HLS versions available if private.
Only the adp streams, so if you untick the low quality video setting in extension it may detect something.
It might not be the quality your after though.

**Comment by diegodlh** [comment]
Any news about this?

I was using similar extension ["Easy Youtube Video Downloader Express"](https://www.yourvideofile.org/) and it also stopped working for private videos some time ago.

I tried with [yt-dlp](https://github.com/yt-dlp/yt-dlp) (using --cookies-from-browser setting) and it's working ok.
